react-challenges/github-finder/devashmit/src/index.css

```
@@ -0,0 +1,365 @@
+.stat-box__label {
+  display: block;
+  font-size: 0.75rem;
+  color: #8b92b8;
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+  margin-top: 0.2rem;
+}
+
+/* ===== Repo List ===== */
+.repo-list__title {
+  font-size: 1rem;
```

Stylelint (17.12.0)

[error] Duplicate property "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)